Onboarding/Connector/Scripts/Documentation/PoweshellScriptTsgAndFaq.md

```diff
@@ -68,7 +68,7 @@ Or
 
 3. The results should look like below, with a row for the connector resource
 
-![Viewing the Connector in ARG](Images/Viewing the Connector in ARG.png)
+!![Viewing the Connector in ARG](Images/Viewing the Connector in ARG.png)
```

Comment:
This has an extra !.
The '%20' should be added for spaces like the other links.

Reply:
Done